src/plugins/remoteClock/requestInterceptor.js

```diff
@@ -31,10 +31,14 @@ function remoteClockRequestInterceptor(openmct, _remoteClockIdentifier, waitForB
       return activeClock?.key === 'remote-clock' && !remoteClockLoaded;
     },
     invoke: async (request) => {
-      const { start, end } = await waitForBounds();
-      remoteClockLoaded = true;
-      request.start = start;
-      request.end = end;
+      const timeContext = request?.timeContext ?? openmct.time;
+      //if we're not in realtime mode, we will use the fixed time bounds provided by the request
+      if (timeContext.isRealTime()) {
```

Comment:
Should this be `if (!timeContext.isRealTime())` ?

Reply:
No. We only want to take the bounds from `waitForBounds()` if we're in realtime mode

Reply:
Can we clarify the comment? The way it reads seems confusing.

Reply:
Yeah, I agree, I think this is good to go after making this comment more clear.